modules/creating-custom-ssl-certs-config-bundle.adoc

```diff
@@ -0,0 +1,238 @@
+:_content-type: PROCEDURE
+[id="creating-custom-ssl-certs-config-bundle"]
+= Creating a custom SSL/TLS configBundleSecret resource
+
+After creating your custom SSL/TLS certificates, you can create a custom `configBundleSecret` resource for {productname-ocp}, which allows you to upload `ssl.cert` and `ssl.key` files.
+
+.Prerequisites
+
+* You have base64 decoded the original config bundle into a `config.yaml` file.
+* You have generated custom SSL certificates and keys.
+
+.Procedure
+
+. If not already set, update your `config.yaml` file to include the `PREFERRED_URL_SCHEME: https`, `EXTERNAL_TLS_TERMINATION: false`, and `SERVER_HOSTNAME` fields:
```

Comment:
`PREFERRED_URL_SCHEME: https`, `EXTERNAL_TLS_TERMINATION: false` is not necessary, it only must route:unmanaged